### PR TITLE
Completed prep-css-flexbox exercise - Andrew Almeida

### PR DESCRIPTION
**Pull request by A-MAlmeida**
030021A5-8880-4096-AE18-F4C930D64DAF_1_105_c
Screen Shot 2021-08-06 at 9 02 06 PM
Screen Shot 2021-08-06 at 9 02 21 PM
Screen Shot 2021-08-06 at 9 02 30 PM

### CODE REVIEW

File: prep-css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
```
Comment (thebearingedge) [inline prep-css-flexbox/display-flex.html:12]: Hi Andrew, looks like your HTML files didn't get save/added in the right order. Be sure to save them in VS Code before doing the `git add` step.

https://github.com/A-MAlmeida/prep-0721-code-solutions/pull/8/files

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by A-MAlmeida** [comment]
B00A9E11-CF2C-4447-A06C-1F7B5FCB4281_1_105_c

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Andrew, somethings up. For some reason your HTML files are missing most of their CSS

https://github.com/A-MAlmeida/prep-0721-code-solutions/pull/8/files

**Comment by A-MAlmeida** [comment]
D34C9A58-79B2-465B-A7EA-B02033D319C2_1_105_c
BB219564-D0A6-40B2-AFE2-B496CBDBD40B_1_105_c
9BE87761-BA30-4261-A5B8-81058BA998F7_1_105_c
9C529BDF-6055-430B-A9E4-98FD63F1B183

**Review by thebearingedge — APPROVED** [review]
Got it!

**Pull request merged**